index.js

```diff
@@ -0,0 +1,83 @@
+import fs from 'fs';
+
+const data = fs.readFileSync('words_alpha.txt', 'utf8');
+//console.log(data);
+const words = data.split('\r\n');
+//console.log(words);
+/*let longestWord = words.reduce(function(a, b) { 
+  return a.length > b.length ? a : b
+}, '');*/
+// console.log(longestWord, words.indexOf(longestWord));
+
+
+
+
+
+
+const ALPHABET = ['a', 'b', 'c','d','e','f','g','h','i','j','k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'];
+
+//console.log(isAlphabetical("acbde"))
+
+
+function focusOnWords(arr){ // We pass in an array of words 
+
+    let result;
+    let foundWords = []; //This is the array that'll list the words that got their letters in alphabetical order
+
+    for(let i = 0; i < arr.length; i++){ // We loop through the words inside the array
```

Comment:
check the indentation of the file. for example there should be a space after for and if